src/smc-webapp/project_new.cjsx

```diff
@@ -239,8 +240,11 @@ ProjectNew = (name) -> rclass
         e.preventDefault()
         if @state.filename[@state.filename.length - 1] == '/'
             @create_folder()
-        else
+        else if @refs.project_new_filename.getValue().includes('.')
```

Comment:
Needs changing to new way of getting input values. Search code for `<FormControl`. It's what's replaced `<Input` and no longer has a `getValue()` function.